src/libos/src/net/addr.rs

```diff
@@ -27,8 +30,12 @@ impl AnyAddr {
                 Self::Ipv6(ipv6_addr)
             }
             libc::AF_UNIX | libc::AF_LOCAL => {
-                let unix_addr = UnixAddr::from_c_storage(c_addr, c_addr_len)?;
-                Self::Unix(unix_addr)
+                // TODO: Re-enable untrusted Unix domain socket
```

Comment:
Should we handle it in this PR?

Reply:
This has been handled. Changed the comments.